R/census_vectors.R

```diff
@@ -0,0 +1,221 @@
+#' Query the CensusMapper API for available vectors for a given dataset.
```

Comment:
Which part adds support for the tax releases?

Reply:
New aggregation type and units in list_census_vectors to help with new metadata and variable types.